components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/util/Utils.java

```diff
@@ -1124,4 +1124,21 @@ public static boolean isUseVerifyClaimEnabled() {
         return Boolean.parseBoolean(IdentityUtil.getProperty
                 (IdentityRecoveryConstants.ConnectorConfig.USE_VERIFY_CLAIM_ON_UPDATE));
     }
+
+    /**
+     * Clones the given map.
+     *
+     * @param map          Map.
+     * @return             Cloned Map.
+     */
+    public static Map<String, Object> cloneMap(Map<String, Object> map) {
+        if (map == null) {
```

Comment:
Keep a space after the method declariont

Reply:
Fixed in 61d7548fc71aca19ecb53cb10928da2bb00294d6